From 9223958816f9df133ae936c9371378ba1203e0da Mon Sep 17 00:00:00 2001
From: Namjae Jeon <namjae.jeon@samsung.com>
Date: Fri, 16 Jul 2021 17:16:11 +0900
Subject: [PATCH] ksmbd: fix typo of MS-SMBD
Git-commit: 9223958816f9df133ae936c9371378ba1203e0da
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


Fix typo : "MS-KSMBD" => "MS-SMBD".

Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/ksmbd/transport_rdma.c | 2 +-
 fs/ksmbd/transport_rdma.h | 6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/fs/ksmbd/transport_rdma.c b/fs/ksmbd/transport_rdma.c
index f818fe358f31..f2ae6bae83f1 100644
--- a/fs/ksmbd/transport_rdma.c
+++ b/fs/ksmbd/transport_rdma.c
@@ -58,7 +58,7 @@
 
 /*
  * User configurable initial values per SMB_DIRECT transport connection
- * as defined in [MS-KSMBD] 3.1.1.1
+ * as defined in [MS-SMBD] 3.1.1.1
  * Those may change after a SMB_DIRECT negotiation
  */
 /* The local peer's maximum number of credits to grant to the peer */
diff --git a/fs/ksmbd/transport_rdma.h b/fs/ksmbd/transport_rdma.h
index 72e2574079f3..0fa8adc0776f 100644
--- a/fs/ksmbd/transport_rdma.h
+++ b/fs/ksmbd/transport_rdma.h
@@ -9,7 +9,7 @@
 
 #define SMB_DIRECT_PORT	5445
 
-/* SMB DIRECT negotiation request packet [MS-KSMBD] 2.2.1 */
+/* SMB DIRECT negotiation request packet [MS-SMBD] 2.2.1 */
 struct smb_direct_negotiate_req {
 	__le16 min_version;
 	__le16 max_version;
@@ -20,7 +20,7 @@ struct smb_direct_negotiate_req {
 	__le32 max_fragmented_size;
 } __packed;
 
-/* SMB DIRECT negotiation response packet [MS-KSMBD] 2.2.2 */
+/* SMB DIRECT negotiation response packet [MS-SMBD] 2.2.2 */
 struct smb_direct_negotiate_resp {
 	__le16 min_version;
 	__le16 max_version;
@@ -37,7 +37,7 @@ struct smb_direct_negotiate_resp {
 
 #define SMB_DIRECT_RESPONSE_REQUESTED 0x0001
 
-/* SMB DIRECT data transfer packet with payload [MS-KSMBD] 2.2.3 */
+/* SMB DIRECT data transfer packet with payload [MS-SMBD] 2.2.3 */
 struct smb_direct_data_transfer {
 	__le16 credits_requested;
 	__le16 credits_granted;
-- 
2.33.0

